sdk/identity/Azure.Identity/src/DefaultAzureCredentialFactory.cs

```diff
@@ -31,9 +31,9 @@ public virtual IExtendedTokenCredential CreateSharedTokenCacheCredential(string
             return new SharedTokenCacheCredential(tenantId, username, Pipeline);
         }
 
-        public virtual IExtendedTokenCredential CreateInteractiveBrowserCredential()
+        public virtual IExtendedTokenCredential CreateInteractiveBrowserCredential(string tenantId)
         {
-            return new InteractiveBrowserCredential(null, Constants.DeveloperSignOnClientId, Pipeline);
+            return new InteractiveBrowserCredential(tenantId, Constants.DeveloperSignOnClientId, Pipeline);
```

Comment:
Can we record a test that uses the value?

Reply:
We have mock tests for the `InteractiveBrowserCredential` which use this parameter, and also a manual test which uses it. We can record a test but it's more involved than normal recorded tests as the interactive flow also stands up an HttpListener on a separate port to receive the redirect from the browser, so that would have to be captured and mocked some way as well.